rotkehlchen/chain/evm/decoding/rainbow/decoder.py

```diff
@@ -40,10 +40,9 @@ def __init__(
             evm_inquirer: 'EvmNodeInquirer',
             base_tools: 'BaseDecoderTools',
             msg_aggregator: 'MessagesAggregator',
-            router_address: ChecksumEvmAddress,
     ) -> None:
         super().__init__(evm_inquirer, base_tools, msg_aggregator)
-        self.router_address = router_address
+        self.router_address = RAINBOW_ROUTER_CONTRACT
```

Comment:
why are you assigning a constant to a class variable? That is not needed. If it's constant use it as a constant everywhere.

Because right now you are just replicating this variable 7 times. 1 the already defined constant and 6 times for each decoder. 


```suggestion
```